frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/backing-storage-step/backing-storage-step.tsx

```diff
@@ -238,70 +215,66 @@ export const BackingStorage: React.FC<BackingStorageProps> = ({
-          }
-          id={`bs-${BackingStorageType.EXISTING}`}
-        />
-        <Radio
-          label={t('ceph-storage-plugin~Create a new StorageClass using local storage devices')}
-          description={t(
-            'ceph-storage-plugin~OpenShift Data Foundation will use a StorageClass provided by the Local Storage Operator (LSO) on top of your attached drives. This option is available on any platform with devices attached to nodes.',
-          )}
-          name={RADIO_GROUP_NAME}
-          value={BackingStorageType.LOCAL_DEVICES}
-          isChecked={type === BackingStorageType.LOCAL_DEVICES}
-          onChange={onRadioSelect}
-          isDisabled={hasOCS || deployment === DeploymentType.MCG}
-          id={`bs-${BackingStorageType.LOCAL_DEVICES}`}
-        />
-        <Radio
-          label={t('ceph-storage-plugin~Connect an external storage platform')}
-          description={t(
-            'ceph-storage-plugin~OpenShift Data Foundation will create a dedicated StorageClass.',
-          )}
-          name={RADIO_GROUP_NAME}
-          value={BackingStorageType.EXTERNAL}
-          isChecked={type === BackingStorageType.EXTERNAL}
-          onChange={onRadioSelect}
-          isDisabled={allowedExternalStorage.length === 0 || deployment === DeploymentType.MCG}
-          body={
-            showExternalStorageSelection &&
-            deployment !== DeploymentType.MCG && (
-              <ExternalSystemSelection
-                selectedStorage={externalStorage}
-                dispatch={dispatch}
-                selectOptions={allowedExternalStorage}
-                stepIdReached={stepIdReached}
-              />
-            )
-          }
-          id={`bs-${BackingStorageType.EXTERNAL}`}
-        />
-        {isMCGStandalone && (
-          <AdvancedSection
-            dispatch={dispatch}
-            deployment={deployment}
-            isAdvancedOpen={isAdvancedOpen}
+        {!hasOCS && <SelectDeployment dispatch={dispatch} deployment={deployment} />}
+        <FormGroup
+          label={t('ceph-storage-plugin~Backing storage type')}
+          fieldId={`bs-${BackingStorageType.EXISTING}`}
+        >
+          <Radio
+            label={t('ceph-storage-plugin~Use an existing StorageClass')}
+            description={t(
+              'ceph-storage-plugin~OpenShift Data Foundation will use an existing StorageClass available on your hosting platform.',
+            )}
+            name={RADIO_GROUP_NAME}
+            value={BackingStorageType.EXISTING}
+            isChecked={type === BackingStorageType.EXISTING}
+            onChange={onRadioSelect}
+            isDisabled={hasOCS || sc?.items?.length === 0}
+            body={
+              showStorageClassSelection && (
+                <StorageClassSelection dispatch={dispatch} selected={storageClass} />
+              )
+            }
+            id={`bs-${BackingStorageType.EXISTING}`}
+            className="odf-backing-store__radio--margin-bottom"
```

Comment:
wrapping under form group , had some of its default css, hence added a class to fix that